packages/runtime/datastore-definitions/src/dataStoreRuntime.ts

```diff
@@ -130,4 +141,17 @@ export interface IFluidDataStoreRuntime extends
      * Errors raised by distributed data structures
      */
     raiseContainerWarning(warning: ContainerWarning): void;
+
+    /**
+     * NOTE: Search blob concept!
```

Comment:
Please remove references to search, This should be generic infrastructure.
Examples can use search in their description - no problem here.